packit/schema.py

```diff
@@ -207,6 +208,25 @@ def make_instance(self, data, **kwargs):
         return TestCommandConfig(**data)
 
 
+class LabelRequirementsSchema(Schema):
+    present = fields.List(fields.String)
+    absent = fields.List(fields.String)
+
+    @post_load
+    def make_instance(self, data, **kwargs):
+        return LabelRequirementsConfig(**data)
+
+
+class RequirementsSchema(Schema):
+    """Configuration of test command."""
+
+    label = fields.Nested(LabelRequirementsSchema, missing=None)
```

Comment:
I wonder if it wouldn't be easier implementation-wise to have an empty instance rather than `None`. With regards to pseudocode I would kinda expect some condition like:
```
(require.label.present IS EMPTY OR require.label.present & labels IS NOT EMPTY)
AND (require.label.absent & labels IS EMPTY)
```

for running the job.

That way we could totally avoid the `None`-checks, cause empty sets make sense as the default values.

Reply:
it is initialised later in the `RequirementsConfig.__init__`, see the tests

Reply:
oh right /o forgot it's twice